c2c/code-to-cloud-spec.md

```diff
@@ -242,10 +242,11 @@ Contains container image related properties.
 
 |Identifier   	|Description	                            |Default Value   	 |
 |---	        |---	                                    |---	             |
-|name   	    |Name of the container image                |$USER-$MODULE_NAME  |
-|repository   	|Container repository to host the container |""   	             |
+|name   	    |Name of the container image                |"hello"             |
```

Comment:
Is default image name is `hello` for all the projects?

Reply:
It is mentioned in the [c2c-schema.json](https://github.com/ballerina-platform/module-ballerina-c2c/blob/211bb5d73b8a613d616e327c5f51e0869684f7b0/c2c-extension/src/main/resources/c2c-schema.json#L38). However, I checked with a sample. It doesn't use `hello`.